app/src/main/java/com/kodeco/android/countryinfo/nav/CountryInfoNavHost.kt

```diff
@@ -62,6 +61,14 @@ fun CountryInfoNavHost(repository: CountryRepository){
             }
         }
 
+        composable("settingsScreen"){
+            CountryAppSettingsScreen(
+                viewModel = hiltViewModel()
```

Comment:
Super minor, but I _think_ you can also call the `hiltViewModel` as a default value for the `viewModel` param in the composable screens that have viewmodels.  That way you don't have to call them and reference this method here at the callsite(s).